Makefile

```diff
@@ -270,3 +271,40 @@ dependency_updates:
 
 benchmark_fast_check:
 	gradle-profiler --benchmark --project-dir subprojects --scenario-file gradle/performance.scenarios fastCheck
+
+## Gradle cache node
+internal_publish_gradle_cache_node_image: check-docker-creds
+	docker run --rm \
+		--volume /var/run/docker.sock:/var/run/docker.sock \
+		--volume "$(shell pwd)/ci/docker/gradle-cache-node-github":/build \
+		--env DOCKER_REGISTRY=$(DOCKER_REGISTRY) \
+		--env DOCKER_LOGIN=$(DOCKER_LOGIN) \
+		--env DOCKER_PASSWORD=$(DOCKER_PASSWORD) \
+		${IMAGE_DOCKER_IN_DOCKER} publish_docker_image publish /build
+
+GRADLE_CACHE_NODE_TAG=4c224da9f6
+
+deploy_gradle_cache_node: check-host-env
+	cd ./ci/k8s/gradle-remote-cache && \
+	sed -e 's|GRADLE_CACHE_NODE_HOST|$(GRADLE_CACHE_NODE_HOST)|g' -e 's|NODE_IMAGE|$(DOCKER_REGISTRY)/android/gradle-cache-node-github:$(GRADLE_CACHE_NODE_TAG)|g' github-project.yaml | kubectl apply -f - && \
+	echo "Gradle Cache Node web interface should be available soon here: http://$(GRADLE_CACHE_NODE_HOST)"
+
+delete_gradle_cache_node:
+	cd ./ci/k8s/gradle-remote-cache && \
+	kubectl delete -f github-project.yaml
+
+check-host-env:
+ifndef GRADLE_CACHE_NODE_HOST
```

Comment:
We have also an imperative check - `check_defined`.
I'd like to use only one way to do something or have explanations for both if they are not identical.

Reply:
Thanks, fixed
already forgot about this func

![image](https://user-images.githubusercontent.com/1105133/122741174-5e585100-d28d-11eb-802e-d13bdcda5a51.png)